package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "fndr",
   "version": "0.1.0",
+  "prisma": {
+    "seed": "ts-node/register prisma/seed.ts"
+  },
   "private": true,
   "scripts": {
```

Comment:
Maybe to make life simple we can add a script here: "seed": "npx prisma db seed"

That way your colleagues can run: `npm run seed` 
Also, people tend to look for scripts in this part of the package.json